bay/images/Dockerfile.ci-builder

```diff
@@ -1,40 +1,87 @@
 FROM hashicorp/terraform:light AS terraform
-FROM integratedexperts/ci-builder:latest
+FROM php:7.4-cli-alpine
+ARG AHOY_VERSION=2.0.0
+ARG GOJQ_VERSION=0.11.2
+ARG LAGOON_CLI_VERSION=0.10.0
 
 # Install SDP ci dependencies.
+## Add Python dependencies.
 COPY --from=terraform /bin/terraform /bin/terraform
-RUN apt-get update && \
-    apt-get install python3 python3-distutils python3-pip rsync hub shellcheck -y && \
-    apt-get clean -y
 
-RUN curl -sSL https://github.com/openshift/origin/releases/download/v3.11.0/openshift-origin-client-tools-v3.11.0-0cbc58b-linux-64bit.tar.gz -o /tmp/openshift.tgz && \
-    tar -vxxzf /tmp/openshift.tgz -C /tmp && \
-    mv /tmp/openshift-origin-client-tools-v3.11.0-0cbc58b-linux-64bit/oc /bin/oc && \
-    chmod +x /bin/oc
+RUN apk update \
+    && apk add --no-cache \
+    musl-dev \
+    libffi-dev \
+    openssl-dev \
+    make \
+    gcc \
+    python3 \
+    py3-pip \
+    python3-dev \
+    cargo \
+    git \
+    npm \
+    bash \
+    openssh \
+    composer \
+    patch \
+    docker \
+    docker-compose \
+    ncurses
 
-# Support the python shebang.
-# @DEPRECATED
-RUN ln -s /usr/bin/python3 /usr/bin/python && ln -s /usr/bin/pip3 /usr/bin/pip
+## Install GitHub CLI tool - Hub and it's dependencies.
+RUN apk add --no-cache \
+    util-linux \
+    groff \
+    go \
+    make
+
+RUN git clone \
+    --config transfer.fsckobjects=false \
+    --config receive.fsckobjects=false \
+    --config fetch.fsckobjects=false \
+    https://github.com/github/hub.git
+
+RUN cd hub && make install prefix=/usr/local
 
 # Manage python deps via pip rather than apt.
 # ansible via apt is 2.7.7 rather than 2.8+
```

Comment:
Is this comment still valid given this image is on alpine rather than ubuntu now?

Reply:
❌ - I'll push an update.